src/web/components/menu/__tests__/Menu.test.jsx

```diff
@@ -0,0 +1,183 @@
+/* SPDX-FileCopyrightText: 2025 Greenbone AG
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import {afterEach, describe, expect, test, testing} from '@gsa/testing';
+import React from 'react';
+import {rendererWith, screen} from 'web/utils/Testing';
+import Menu from 'web/components/menu/Menu';
+
+testing.mock('web/hooks/useTranslation', () => ({
+  default: () => [key => key],
+}));
+
+afterEach(() => {
+  testing.clearAllMocks();
+});
+
```

Comment:
Mocking the useTranslation hook should not be necessary because by default it used the en locale which just does the same as your mock and returns the key.
```suggestion
```